stage3/ogmios.tf

```diff
@@ -90,5 +90,95 @@ module "ext_cardano_ogmios" {
         }
       ]
     }
+    "instance2" = {
+      salt    = "samplesalt"
```

Comment:
Should this have a different value?

Reply:
Salt now comes in from the provisioning event, so this does nothing, AFAIK.